src/otx/algo/detection/heads/base_head.py

```diff
@@ -468,3 +469,210 @@ def _bbox_post_process(
             results = results[: cfg.max_per_img]
 
         return results
+
+    def export(
+        self,
+        x: tuple[Tensor],
+        batch_data_samples: list[InstanceData],
+        rescale: bool = False,
+    ) -> list[InstanceData]:
```

Comment:
I admit that implementing every export function requires lots of engineering cost. I think we should re-visit this after migrating all other detectors. Currently I add these export function to compare predict procedure more explicitly. And this approach can protect predict function so that we can assure backward compatibility between PRs.
In addition if we use torch.onnx.is_in_onnx_export() branch, then direct export through ov, not using onnx, cannot enter that branch.